logging_manager/src/main.py

```diff
@@ -0,0 +1,45 @@
+# Library imports
+import requests
+import time
+import glob
+import os
+
+# Subtask imports
+from StorageManager import storage_manager_main
+from WiFiUploader import wifi_uploader_main
+
+# Run StorageManager
+storage_result = storage_manager_main()
```

Comment:
Everything from this line down should exist inside a function, this is outside in the global scope